README.md

```diff
@@ -179,6 +197,35 @@ fans:
     curve: cpu_curve
 ```
 
+#### NVIDIA
+
+To use detected NVIDIA GPUs in your configuration, use the `nvidia` fan type:
+
+```yaml
+fans:
+  - id: gpufan1
+    nvidia:
+      # A regex matching a nvidia device as displayed by `fan2go detect`
+      # the following matches all nvidia devices in your system
+      # (good enough if you only have one), otherwise you could
+      # also use nvidia-10DE2489-0800 or similar
+      device: nvidia-
```

Comment:
any reason to include the "trailing minus" if its supposed to match any device anyway?